### PR TITLE
[RLlib Contrib] APEX DQN

### PR DESCRIPTION
**Pull request by avnishn**
Signed-off-by: Avnish <avnishnarayan@gmail.com>

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib_contrib/a3c/pyproject.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ version = "0.1.0"
 description = ""
 readme = "README.md"
 requires-python = ">=3.7, <3.11"
-dependencies = ["gym[accept-rom-license]", "gymnasium[mujoco]==0.26.3", "higher", "ray[rllib]==2.3.1"]
+dependencies = ["gym[accept-rom-license]", "gymnasium[mujoco]==0.26.3", "ray[rllib]==2.3.1"]
```
Comment (ArturNiederfahrenhorst) [inline rllib_contrib/a3c/pyproject.toml:15]: According to our requirements, higher is used for MAML on PyTorch. Not sure why it went here in the first place.
I have no idea what the library does, just wanna point out this irregularity.

File: rllib_contrib/apex_dqn/examples/apex_dqn_cartpole_v1.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import argparse
```
Comment (ArturNiederfahrenhorst) [inline rllib_contrib/apex_dqn/examples/apex_dqn_cartpole_v1.py:1]: why "v1"?

**Review by ArturNiederfahrenhorst — APPROVED** [review]
@avnishn Can you change the name of the example if it is a mistake? Otherwise good to go imo :)

File: rllib_contrib/apex_dqn/examples/apex_dqn_cartpole_v1.py (hunk 2 above)
Comment (avnishn) [reply rllib_contrib/apex_dqn/examples/apex_dqn_cartpole_v1.py:1]: we are using the cartpole v1 environment

File: rllib_contrib/a3c/pyproject.toml (hunk 1 above)
Comment (avnishn) [reply rllib_contrib/a3c/pyproject.toml:15]: It slipped through

**Pull request merged**